inference.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from model import Neural_Network
+from utils import load_weights
+
+def inference(input_data):
```

Comment:
add a doc string, define the return type, add input parameter type

Reply:
doc strings are added.